### ISSUE TITLE
Error summary and success banners - announced twice with some screen readers

### ISSUE THREAD

**Issue by elceebee** [DFE-Digital/govuk_design_system_formbuilder#605]
We have recently had an Accessibility Audit on our product (Apply and Manage). We user the error summary component in a perfectly usual way, in line with the documentation. 

The auditors describe the following problem:

"When navigating with screen readers Android TalkBack or JAWS, form validation errors across the site ... are announced multiple times when a form is submitted with empty fields. Error messages should be announced only once to avoid confusion."

And

"Unique to Android TalkBack, when banners are displayed (success banner, and error messages), only the success state is announced, while the accompanying banner text is not announced."

This does not comply with WCAG Level AA 2.2:
4.1.3 Status Messages – [https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html)

**Comment by peteryates** [comment]
Closing this as we're following the Design System's suggested format for error summaries and success banners.
